cime_config/allactive/config_compsets.xml

```diff
@@ -330,6 +330,14 @@
   <lname>1850_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_MALI%STATIC_SWAV</lname>
 </compset>
 
+<!-- EAMXX fully coupled compset -->
+
+<compset>
+  <alias>WCYCLXX2010</alias>
```

Comment:
I chose this name arbitrarily, so I am open to changing it.

Reply:
@golaz and @rljacob  - do you have an opinion about how we name coupled compsets using EAMxx?

Reply:
We want do distinguish between a WCYCL case with EAM and one with EAMXX.  I think adding "XX" somewhere makes sense.